activerecord/lib/active_record/tasks/database_tasks.rb

```diff
@@ -160,16 +160,17 @@ def setup_initial_database_yaml
         end
       end
 
-      def for_each(databases)
+      def for_each(databases, options={})
```

Comment:
I don't think this change is necessary to implement this feature. I'd rather not have options passed to `for_each`

Reply:
yes, I wasn't so sure about it as well... the thing is, if we don't do this we are gonna add a bit of complexity directly into the rake tasks, since we'll need something very similar to this `for_each` but for the case where there is only one database...

gonna try to think of something else to avoid these problems

Reply:
I couldn't figure out anything interesting here, so I removed the option from the method as you asked and went with the conditional directly into the rake task. 

Let me know if you have anything better in mind!